capture/noworkflow/now/vis/views.py

```diff
@@ -77,6 +79,39 @@ def trials():
                       summarize=bool(int(request.args.get("summarize"))))
     return jsonify(**history.graph.graph())
 
+#generate dafalowdataflow
+@app.route("/trials/<tid>/flow.pdf")
+def dataflow(tid):
+    """Generates the dafalow of a trial """ 
+    trial = Trial(tid)
+    dest = 'flow.pdf'
+    temp = 'flow.dot'
+    # args = ['/bin/sh',"/usr/local/lib/python3.5/dist-packages/noworkflow-2.0.0a0-py3.5.egg/noworkflow/now/vis/static/mydataflow.sh",os.getcwd(),str(tid),temp,dest] 
+    #ret = subprocess.Popen(args)
+    cmd = 'cd ' + os.getcwd()+ ' && now dataflow ' + str(tid) + ' >' + temp +' && dot -T pdf ' + temp + ' -o ' + dest
```

Comment:
Either call a function from noWorkflow, or invoke `now` using `sys.executable`. Using the system `now` may invoke the wrong installation.